internal/scene/sceneDemo.go

```diff
@@ -30,7 +30,7 @@ func (s *Scene) InitDemoScene() {
 	var trackPath = "../assets/"
 	s.trackMap = make(map[int]*effects.Volume)
 	for index, element := range trackArray {
-		fmt.Println(index, trackPath, element)
+		//fmt.Println(index, trackPath, element)
```

Comment:
Did you leave that debug statement in there on purpose?

Reply:
We can leave the whole loop as a comment, this was just way back when I needed to check if the tracks are actually getting added to the array. There is no need to have it printed out now.